semgrep/semgrep/cli.py

```diff
@@ -134,6 +134,9 @@ def cli() -> None:
         help="Apply the autofix patches. WARNING: data loss can occur with this flag. Make sure your files are stored in a version control system.",
         action="store_true",
     )
+    output.add_argument(
```

Comment:
Do we want this to be in a separate file from the config?

Reply:
I don't think this is wired to anything so can probably do without a cli arg for now?

Reply:
Oh, yeah, I forgot to remove it. 😅